packages/components/src/mobile/inserter-button/index.native.js

```diff
@@ -41,10 +42,7 @@ class MenuItem extends Component {
 			styles.modalIconWrapper,
 			styles.modalIconWrapperDark
 		);
-		const modalIconStyle = getStylesFromColorScheme(
-			styles.modalIcon,
-			styles.modalIconDark
-		);
+		const modalIconStyle = styles.modalIcon;
```

Comment:
`modalIconStyle` no longer has a dark mode variant because the fill color is now set by default in the `BlockIcon` component (related to [this comment](https://github.com/WordPress/gutenberg/pull/34967/files#r712886224)).